app/build.gradle.kts

```diff
@@ -11,6 +11,12 @@ configurations.all {
 dependencies {
     implementation(project(":read-db-core"))
 
+    //region extensions
+    runtimeOnly(project(":read-db-oracle-extension")) {
```

Comment:
note: I believe it would be better to group this exception with the driver. I grouped drivers because they were the only dependencies. Now, when we have extensions I think we should group the driver and its extension